recipes/mpl-scatter-density/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "mpl-scatter-density" %}
+{% set version = "0.2" %}
+{% set sha256 = "c1356e2e610141569951ffa2ecde271504f4af321447e53b7bc59f2a8681b7ee" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
```

Comment:
`fast-histogram` is only available for `np111, np112, and bp113` and conda is not smart enough to give you that. So you can use here `numpy >=1.11`. Or, you can try something like [this](https://github.com/ocefpaf/netcdf4-feedstock/blob/594b4056834c38a9f60f4dc4b2eed82ab28943c5/recipe/meta.yaml) in `fast-histogram`.